network/imds/imds_windows.go

```diff
@@ -29,19 +29,22 @@ const (
 	hnsAclPolicyHighPriority = 200
 	// hnsAclPolicyLowPriority represents the lower priority number.
 	hnsAclPolicyLowPriority = 300
+	// instanceMetadataEndpoint represents the IMDS endpoint over IPv4
+	// We will ideally want to use vpc.InstanceMetadataEndpoints once we can block IPv6 endpoint with HNS
+	instanceMetadataEndpointV4 = "169.254.169.254/32"
```

Comment:
Can we please move this definition to right after the vpc.InstanceMetadataEndpoints definition in vpc/vpc.go? Having both metadata endpoint definitions in the same place will ensure the next person who changes them to notice that Linux and Windows use different endpoint definitions and take that into account when making changes.